apps/examples/testcase/le_tc/drivers/tc_pwm.c

```diff
@@ -0,0 +1,156 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file tc_pwm.c
+/// @brief Test Case Example for pwm driver
+#include <tinyara/config.h>
+#include <tinyara/pwm.h>
+#include <stdio.h>
+#include <fcntl.h>
+#include "tc_internal.h"
+
+/**
+* @fn                   :tc_pwm_open_close
+* @brief                :Test the pwm driver open and close
+* @scenario             :Test the pwm driver open and close
+* API's covered         :open, close
+* Preconditions         :none
+* Postconditions        :none
+* @return               :void
+*/
+static void tc_pwm_open_close(void)
+{
+	int fd = 0, fd2 = 0;
+	int ret = 0;
+
+	fd = open("/dev/pwm0", O_RDWR);
+	TC_ASSERT_GT("pwm_open", fd, 0);
+
+	fd2 = open("/dev/pwm0", O_RDWR);
+	TC_ASSERT_GT_CLEANUP("pwm_open", fd, 0, close(fd));
```

Comment:
why are you checking `fd `? isn't it `fd2`?

Reply:
Yes. Done